docs/datetime.adoc

```diff
@@ -6,11 +6,8 @@ endif::env-guide[]
 
 == Conversion between formatted dates and timestamps
 
-* `apoc.date.parseDefault('2015-03-25 03:15:59','s')` get Unix time equivalent of given date (in seconds)
 * `apoc.date.parse('2015/03/25 03-15-59','s','yyyy/MM/dd HH/mm/ss')` same as previous, but accepts custom datetime format
```

Comment:
mark that here everything beyon the first parameter is optional .e.g with `['s']`